tests/PhpUnit/JsonSchemaConstraintTest.php

```diff
@@ -53,7 +54,8 @@ public function testValidSchema()
 JSON;
         $response = json_decode($response);
 
-        self::assertTrue($this->constraint->evaluate($response, '', true), $this->constraint->evaluate($response));
```

Comment:
because now we don't use a new validator each time matches() is called, and because of the validator states the errors are duplicated.

And the absence of exception means the test is a success.